src/Service/AlertManager.php

```diff
@@ -263,12 +263,13 @@ public function formatAlert(NodeInterface $alert) {
 
     return [
       'title' => $alert->getTitle(),
-      'textColor' => $alert->field_alert_text_color->entity->field_color->value,
-      'bgColor' => $alert->field_alert_color->entity->field_color->value,
-      'description' => $alert->field_alert_description->value,
+      'textColor' => $alert?->field_alert_text_color?->entity?->field_color?->value,
+      'bgColor' => $alert?->field_alert_color?->entity?->field_color?->value,
```

Comment:
shouldn't we use fallback default color in this case?

Reply:
@podarok This case is already handled on the js side:
 https://github.com/open-y-subprojects/openy_node_alert/blob/c8bb70042fb55fe1862334440d9bc434255690e8/js/openy-alerts/src/components/HeaderAlertItem/HeaderAlertItem.js#L67
 https://github.com/open-y-subprojects/openy_node_alert/blob/c8bb70042fb55fe1862334440d9bc434255690e8/js/openy-alerts/src/components/HeaderAlertItem/HeaderAlertItem.js#L89